Spanner/tests/System/SpannerTestCase.php

```diff
@@ -117,4 +117,11 @@ public static function getDatabaseInstance($dbName)
 
         return self::$client->connect(self::INSTANCE_NAME, $dbName);
     }
+
+    public function skipEmulatorTests()
```

Comment:
```suggestion
    public static function skipEmulatorTests()
```

Static methods in PHP can be called using `::` or `->`.

Reply:
Good to know! :+1: